### DISCUSSION TITLE
Shrinking Map Alternative:

### DISCUSSION THREAD

**Discussion by aeldred28** [FakeFishGames/Barotrauma#5086] [Ideas and suggestions]
_I've posted this on the steam suggestion page as well, this is the topic;_

When I ask around, I can't find any community members who are actually excited from what they've seen of the shrinking map feature, in-which the map has radiation that forces the player to the move on.

As I understand it, the purpose of this feature is to prevent the player backtracking or remaining in one zone forever. I think this solution is fundamentally counterproductive to the gameplay experience, and I've thought of a less complicated solution to consider:

**Suggestion: Instead of implementing intrusive radiation to the map that doesn't make sense to the game-world, and doesn't add substantive value to the experience -- solve the problem of player backtracking/progression by making it so that missions only ever direct the player to the right side of the map by spawning missions in-either the upward, downward, or rightward location connections. Currently the player can get missions that send you all the way back to the start of the game chasing money. And it is annoying regardless that good missions keep sending you back to places you have already been.**

By making it so that missions direct the player rightward, there are arguably equal consequences for the player choosing to backtrack or hang out in a single zone compared to the current shrinking-map feature. Fuel-death, or simply running out of resources is the risk in backtracking here. As there are no missions that would direct the player leftward, the player would have to do so at a cost/risk to themselves. This solution does not raise any questions about the lore, nor does it add a direct annoyance to the player's campaign. It is purely a subconscious solution to backtracking.

Even with the shrinking-map, making it so missions don't spawn backwards is a worthwhile fix. As being forced to leave a fun/rewarding missions behind because the player was forced to flee radiation is frustration I doubt any player wants.

Was wondering if the devs would consider, or offer a response to this idea,

-Scale

**Comment by Dynamoon** [comment]
Hi! Thanks for the feedback, seeing it made us realise we should probably clarify our plans a little. I just replied to you on the Steam forums, moments before I noticed you'd posted here too, so I'll copypaste the response I wrote with a few other teammates:

"As I understand it, the purpose of this feature is to prevent the player backtracking or remaining in one zone forever. I think this solution is fundamentally counterproductive to the gameplay experience"

This is a bit of an over-simplification, to our eyes. We've heard many players anticipate an overtly FTL-like map solution when they hear "shrinking map", and that's not exactly what we have planned either; we find the FTL map a little too aggressive for our purposes and aren't looking to replicate that exactly with Barotrauma. So while we want to add a new, hard-to-argue-with incentive to move forward (and as a side note, it ties in with the lore rather than being something we just made up for this feature! ;) ), it's not done so much to turn the game into a race where you must carefully plan every step, but simply to prevent farming the lower levels for resources, skills and upgrades. It's currently a little too easy to just work your way around the intended increase in difficulty by doing that.

I don't think we'll be fundamentally changing how we implement the new map. However, having seen a lot of apprehension regarding this change, we are considering making it optional – we like a fair bit of sandboxiness, and it's not the purpose of our work to force a particular playstyle on anyone, and it's starting to look like here we might risk doing just that.

I realise this is still pretty vague in terms of details, but I hope it clears up our intentions with this feature a little! Once the shrinking map is released into the unstable version, we look forward to hearing what you and everyone else thinks about how it works in practice.

**Reply by Xeavas** [reply]
Chipping in with some suggestions relating on how to make the radiation not overly intrusive and helps smooth the difficulty curve. TLDR; Make radiation stop advancing at a certain point and slightly increase the difficulty to make a more gradual transition from the plateau to latter biomes.
(picture below)
![radsuggestions](https://user-images.githubusercontent.com/72674994/108718273-395f0600-7559-11eb-89e5-7e717f729172.jpg)

**Comment by ubertpendragon** [comment]
Radiation doesn't really make sense thematically since outposts settle as you progress and clear areas, so there has to be an alternative

Rather than danger pushing you, it should be the lack of opportunity; "civilization". The more colonies progress, the less opportunity there should be: 
![unknown](https://user-images.githubusercontent.com/73229309/108109528-4d56c380-7092-11eb-9089-1c054fb3079e.png)


You are a frontier captain paid for dangerous jobs. That's your calling, so you have to keep pushing forward

**Comment by Regalis11** [comment]
That's a pretty good idea too! Although I have to say the radiation does have a thematical purpose and ties in (or rather, will tie in) with the game's story and ending. ;) I unfortunately can't say much more about that without spoiling it.

But in any case, I hope you'd wait for the feature to be out in the unstable version before giving feedback on it. It seems you may be overestimating how fast the radiation advances or how much of an effect it has on the gameplay. If you play the game "normally" it most likely won't affect your campaign much, but if you spend so much time farming resources in the initial biomes that it'd make the rest of the campaign way too easy, the radiation will slowly start to have an effect. And as for "rather than danger pushing you, it should be the lack of opportunity", I see the radiation doing precisely the latter. It slowly reduces the incentives to stay in the initial areas and makes it more costly, it's not the kind of "wall of instant game over" as in FTL.

**Reply by ubertpendragon** [reply]
> it's not the kind of "wall of instant game over" as in FTL.

The fleet catching up isn't instant game over in FTL, that's the beauty of it! But to understand, you'll have to check  #5129 out ;)

**Comment by Benzin73** [comment]
Hello!
Ibe allready made a post in the steam "Development Suggestions" forum about this
https://steamcommunity.com/app/602960/discussions/2/3113646913573869453/
But I tought id give my 2c here as well.
Please note, this is from the perspective of someone who only does single player.

I must say this idea of forcing us to keep moving does not apeal to me, **at all**. And i do hope that if this feature ends up being implemented, there will be an option to completely disable it. At least for single player campaign.

I love playing the game. Why do you want to rush me to finish it quickly? :)
I enjoy roaming arround all areas. Taking my time, exploring the awesome new caves you guys implemented, doing missions, going back to outposts to get someone / something i saw earlier, etc. And forcing me to "keep moving right, or radiation will kill you!" will completely destroy the experience (for me). If i may be completely honest, this would in fact be a "game killer" for me.
Please give us the option to play the game at the pace we enjoy.

Thanks!

**Comment by Udrakan** [comment]
With your proposal, you could go one node to the right with a mission, go back left, then go right with a mission again.

**Reply by aeldred28** [reply]
Technically you could do that anyways. The difference here is that doing that comes at a cost to your campaign. You won’t get missions that take you straight backwards.

**Reply by Ek-Videogames** [reply]
The problem is you **have** to grind missions in certain areas in order to afford change of submarine or upgrades or to stock on supplies before going to the abyss. Also making outpost/faction reputation & outpost evolution not matter at all in the grand scheme of things.
If you get unlucky mission options you can through no fault of your own be softlocked at the edge of the abyss with not enough supplies/upgrades.

I'd settle for whatever solution is found that in some way addresses the problems with campaign economy balance concern.
